yabeda-rails.gemspec

```diff
@@ -24,7 +24,8 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.4"
 
-  spec.add_dependency "rails"
+  spec.add_dependency "activesupport"
+  spec.add_dependency "railties"
```

Comment:
Not sure what is the real use-case for replacing dependencies here. In its current version this gem doesn't make sense without ActionController (which is included into `actionpack`) and (optionally) without `actionview` and `activerecord`.

Reply:
Railties depends on actionpack but thus also actionview, but activerecord is optional.